packages/syft/src/syft/client/client.py

```diff
@@ -762,6 +762,26 @@ def connect(
     return _client
 
 
+@instrument
+def register(
```

Comment:
@kiendang can you also add the register method to `packages/syft/src/syft/__init__.py`

Reply:
oh right sorry